README.md

```diff
@@ -25,7 +25,13 @@ This is a work in progress section. Currently, in order to be able to use the bl
 
 ```sh
     cd $ZEPHYR_BASE
-    west build -b bloodlight_rev2 <bloodlight-zephr directory path> -- -DBOARD_ROOT=<bloodlight-zephr directory path>
+    west build <bloodlight-zephr directory path/${Desired app}>
+```
+
+If you would like to make a clean build use
+
+```sh
+    west build -p always <bloodlight-zephr directory path/${Desired app}>
```

Comment:
Don't we need to specify which board to use with west?

Reply:
It's not necessary because it's already specified in the Cmake. 

https://github.com/CodethinkLabs/bloodlight-zephyr/blob/df25a7ba00e42f00c24ae5dfebeee4fa34231f3e/CMakeLists.txt#L7

https://github.com/CodethinkLabs/bloodlight-zephyr/blob/df25a7ba00e42f00c24ae5dfebeee4fa34231f3e/CMakeLists.txt#L8